src/components/Home/BookingFloatingButton.tsx

```diff
@@ -1,23 +1,27 @@
 import { useNavigate } from 'react-router';
+import { useRecoilValue } from 'recoil';
 import styled from 'styled-components';
 
-interface BookingFloatingButtonProps {
-  selectedCard: boolean;
-}
+import { movieInfoState } from '../../recoil/atom';
 
-const BookingFloatingButton = ({ selectedCard } : BookingFloatingButtonProps) => {
+const BookingFloatingButton = () => {
   const navigate = useNavigate();
+  const movieId = useRecoilValue(movieInfoState).movie_id;
 
   const handleBtn = () => {
-    navigate('/select-time');
+    if (movieId !== 0) navigate('/select-time');
+    console.log(movieId);
+    return movieId;
   };
 
   return (
     <St.ButtonWrapper>
       <St.ButtonContents
         type='button'
-        onClick={handleBtn}
-        disabled={selectedCard}
+        onClick={() => {
+          handleBtn();
+        }}
+        disabled={(movieId === 0) ? true : false}
```

Comment:
```suggestion
        disabled={(movieId === 0)}
```

그냥 요렇게만 해줘도 true / false로 들어갈것 같네요!